HeadsDown/AppDelegate.swift

```diff
@@ -27,7 +27,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         UserPreferences.isEnabled = true
         DoNotDisturb.isEnabled = false
         
-        //queryApps()
+        NSApp.activate(ignoringOtherApps: true)
```

Comment:
What does this do?

Reply:
Brings window to the forefront when its opened.